src/routes/{{.Route}}.js

```diff
@@ -1,18 +1,28 @@
 import { Router } from "express";
 import { getData } from "../services/dataHandler";
 import ExampleError from "../errors/ExampleError";
+import Jaeger from "./jaeger";
+import CircuitBreaker from "opossum";
 
 const router = Router();
 
+const opossumOptions = {
+  timeout: 15000, // If our function takes longer than 10 seconds, trigger a failure
+  errorThresholdPercentage: 50, // When 50% of requests fail, trip the circuit
+  resetTimeout: 30000, // After 30 seconds, try again.
+};
+
 /**
  * GET /api/v1/{{.Route}}
  * @description Example route
  * @response 200 - OK
  * @response 400 - Error
  */
 router.get("/", async (req, res, next) => {
+  const jaegerTracer = new Jaeger({{.Route}}, req, res);
```

Comment:
I think we wanted to call this context, but jaegerTracer is prolly fine too @moficodes what do you think?

Reply:
Will rename it to context

Reply:
Done